packages/experiment-tag/src/subscriptions.ts

```diff
@@ -373,6 +378,62 @@ export class SubscriptionManager {
     });
   };
 
+  private setupExitIntentPublisher = () => {
+    // Check if any page objects use exit_intent trigger
+    const hasExitIntentTriggers = Object.values(this.pageObjects).some(
```

Comment:
I'm not familiar with the flow but does this `pageObjects` already filter only the active page objects?

nit: if you're using `pages` multiple times, you could assign it at the top first:
```ts
const pages = Object.values(this.pageObjects).flatMap((pages) => Object.values(pages));
```